### PR TITLE
Updated pipeline status links and refactored gh-pages branch

### PR DESCRIPTION
**Pull request by Jonsy13**
Signed-off-by: Jonsy13 <vedant.shrotria@chaosnative.com>

**This PR will -** 
- Update the pipeline status links
- Refactor the pipeline-runs files ( Made a `common` directory for `function.js` )
- Update the Readme.md
- Delete the Github-Action Workflows

**Demo GH-Pages deployed Here** - https://jonsy13.github.io/Pipeline-Updates-Test/index

### CODE REVIEW

File: portal-pipeline/pipeline-runs/portal-rc.html [hunk 1]
```diff
@@ -10,14 +10,24 @@
...
       <h1 class="project-name">Litmus-Portal E2E Runs</h1>
-      <h2 class="project-tagline"><b>RC builds for Litmus-Portal E2E</b></h2>
+      <h2 class="project-tagline"><b>RC builds for Portal-E2E</b></h2>
     </section>
     <section class="main-content">      
-    <h2><b>Litmus-Portal RC Build Details</b></h2>
-    <p>Here are the details of latest few pipeline runs for Litmus-Portal tests!</p>  
\ No newline at end of file
+    <h2><b>RC builds for Portal-E2E</b></h2>
+    <p>Here are the details of latest few pipeline runs for Portal Tests!</p>
```
Comment (oumkale) [inline portal-pipeline/pipeline-runs/portal-rc.html:27]:
```suggestion
    <p>Here are the details of the recent pipeline runs for Portal Tests!</p>
```

File: portal-pipeline/pipeline-runs/portal-run.html [hunk 2]
```diff
@@ -10,96 +10,24 @@
...
- <tr>
-  <td><a href= "https://gitlab.mayadata.io/litmuschaos/litmus-e2e/pipelines/13557">13557</a></td>
-  <td>Sun Mar 28 00:00:10 2021(IST)</td>
-  <td>ci</td>
- <td><a href="https://bit.ly/2OLie8t"><img alt=60% src="https://progress-bar.dev/60" /></a></td>
- </tr>
-</table>
\ No newline at end of file
+    <h2><b>Nightly builds for Portal-E2E</b></h2>
+    <p>Here are the details of latest few pipeline runs for Portal Tests!</p>
```
Comment (oumkale) [inline portal-pipeline/pipeline-runs/portal-run.html:27]: Same

**Review by oumkale — APPROVED** [review]

File: portal-pipeline/portal.md [hunk 3]
```diff
@@ -8,21 +8,18 @@ filename: portal
...
-- [TCID-EKS-Portal-GUI-Routes](https://github.com/Jonsy13/litmus-e2e/blob/litmus-portal/testcases/TCID-EKS-Portal-GUI-Routes.md)
-- [TCID-EKS-Portal-GUI-Teaming](https://github.com/Jonsy13/litmus-e2e/blob/litmus-portal/testcases/TCID-EKS-Portal-GUI-Teaming.md)
-- [TCID-EKS-Portal-GUI-User-Management](https://github.com/Jonsy13/litmus-e2e/blob/litmus-portal/testcases/TCID-EKS-Portal-GUI-User-Management.md)
+- [TCID-EKS-Portal-GUI-AccountSettings](https://github.com/Jonsy13/litmus-e2e/blob/gh-pages/testcases/TCID-EKS-Portal-GUI-AccountSettings.md)
+- [TCID-EKS-Portal-GUI-Auth](https://github.com/Jonsy13/litmus-e2e/blob/litmus-portal/gh-pages/TCID-EKS-Portal-GUI-Auth.md)
+- [TCID-EKS-Portal-GUI-Routes](https://github.com/Jonsy13/litmus-e2e/blob/litmus-portal/gh-pages/TCID-EKS-Portal-GUI-Routes.md)
+- [TCID-EKS-Portal-GUI-Teaming](https://github.com/Jonsy13/litmus-e2e/blob/litmus-portal/gh-pages/TCID-EKS-Portal-GUI-Teaming.md)
+- [TCID-EKS-Portal-GUI-User-Management](https://github.com/Jonsy13/litmus-e2e/blob/litmus-portal/gh-pages/TCID-EKS-Portal-GUI-User-Management.md)
```
Comment (uditgaurav) [inline portal-pipeline/portal.md:21]: can we replace `Jonsy13` with `litmuschaos`

**Comment by uditgaurav** [comment]
can we update the adoptor image as well?

**Review by uditgaurav — APPROVED** [review]

**Comment by uditgaurav** [comment]
Thanks for updating the e2e website @Jonsy13 !

**Pull request merged**